Program.cs

```diff
@@ -11,101 +49,96 @@ static Rectangle CellToRectangle(int row, int col, int size)
     }
 
     // trying to draw the apple shape? 
-    static Vector2 CellToCenter(int row, int col, int size)
+    static Vector2 CellToCenter(Cord appleCord, int size)
     {
         // Uhhh, we're converting thaaa indices of the cells into 
         // the centers of where they need to drawn (in the pixel world :o )
-        float centerX = size * (col + 0.5f);
-        float centerY = size * (row + 0.5f);
+        float centerX = size * (appleCord.Y + 0.5f);
+        float centerY = size * (appleCord.X + 0.5f);
         return new Vector2(centerX, centerY);
     }
 
-    static void DrawGrid(int rows, int cols, int size)
+    static void DrawGrid(Grid grid, int size)
     {
-        for (int row = 0; row < rows; row += 1)
+        for (int row = 0; row < grid.cols; row += 1)
         {
-            for (int col = 0; col < cols; col += 1)
+            for (int col = 0; col < grid.rows; col += 1)
```

Comment:
Same as above.

Reply:
Just review these changes everywhere conceptually since I don't think you were disciplined when you did the first replacement.